docs/guide/markdown.md

```diff
@@ -386,7 +386,7 @@ Alternatively, it's possible to highlight directly in the line by using the `//
 export default {
   data () {
     return {
-      msg: 'Highlighted!' // [!code  hl]
+      msg: 'Highlighted!' // [!!code highlight]
```

Comment:
Is this breaking change or `hl` is also supported? 👀

Reply:
https://github.com/antfu/shikiji/blob/a5dacbc8ff703d783182f916c2dc9944ddfd06b6/packages/shikiji-transformers/src/transformers/notation-highlight.ts#L30 -- both are supported.

Reply:
Both are supported, but I think the full name would be clear for ppl to understand.